evolver/controller/standard/chemostat.py

```diff
@@ -16,51 +20,74 @@ class Config(VialConfigBaseModel):
         flow_rate: float = Field(0, description="Flow rate for dilutions")
         stir_rate: float = Field(8, description="Stir rate")
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(self,
+                 od_sensor: HardwareDriver | ConfigDescriptor | str,
+                 pump: HardwareDriver | ConfigDescriptor | str,
+                 stirrer: HardwareDriver | ConfigDescriptor | str,
+                 *args,
+                 vials: list = None,  # TODO: Derive this cls from a VialCentricController and add this there (#30).
+                 window: int = 7,
+                 min_od: float = 0,
+                 start_delay: int = 0,
+                 flow_rate: float = 0,
+                 stir_rate: float = 8,
+                 **kwargs):
+        self._od_sensor = od_sensor
```

Comment:
@amitschang Suggestions on how to handle this?

An option is to obviously add a setter, but then that's a shame since we'd be disallowing properties without setters. We could add a setter that only sets once, i.e., when``self._v is None`` and raise otherwise, but then we'd be baking this into the SDK framework and subsequent developers/users would be forced to do the same.